src/test/java/seedu/address/logic/parser/property/FindPropertiesCommandParserTest.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.parser.property;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.property.FindPropertiesCommand;
+import seedu.address.model.property.PropertyNameContainsSubstringPredicate;
+
+public class FindPropertiesCommandParserTest {
+
+    private FindPropertiesCommandParser parser = new FindPropertiesCommandParser();
+
+    @Test
+    public void parse_emptyArg_throwsParseException() {
+        assertParseFailure(parser, "     ",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindPropertiesCommand.MESSAGE_USAGE));
+    }
+
+    @Test
+    public void parse_validArgs_returnsFindCommand() {
+        // no leading and trailing whitespaces
+        FindPropertiesCommand expectedFindPropertiesCommand =
+                new FindPropertiesCommand(new PropertyNameContainsSubstringPredicate("tembu"));
+        assertParseSuccess(parser, "tembu", expectedFindPropertiesCommand);
+
+        // multiple whitespaces between keywords
+        assertParseSuccess(parser, " \n tembu \n \t \t", expectedFindPropertiesCommand);
```

Comment:
I thought we updated the implementation of `find` to only find the substring? I'm confused as to how this would be a success

Reply:
yup! the `\n tembu \n \t \t` is the user input, so it should strip it to just form `tembu`.

Reply:
just realised \n and \t are new line and tab characters 🥲